From 86bab0d986ef89d634bcca4d538867d708eeec59 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Mon, 22 Feb 2021 09:53:04 -0500
Subject: [PATCH] qemux86: add configuration symbol to select values

On 5.10+ we are seeing some MSI interrupt assignment errors. In
our emulated 32bit platform, we are lacking this support. To avoid
the warnings, we define our own platform and select the MSI arch
fallback, which handles the missing support.

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 arch/x86/Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 4b9f378e05f6..6d3a923d0f3b 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -2979,6 +2979,12 @@ config ISA_DMA_API
 	  Enables ISA-style DMA support for devices requiring such controllers.
 	  If unsure, say Y.
 
+config QEMUX86
+	bool "Enable for qemumx86 specific configurations"
+	default n
+	select PCI_MSI_ARCH_FALLBACKS if PCI_MSI
+
+
 if X86_32
 
 config ISA
-- 
2.39.2

